addons/report/views/layouts.xml

```diff
@@ -75,7 +75,7 @@
             </div>
         </div>
         <div class="row">
-            <div class="col-xs-3">
+            <div class="col-xs-5">
```

Comment:
Yes..., in fact in OCA PR I've done it with 6... but the under line grows too much.